scripts/run-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Cleanup function to stop and remove the container
+cleanup() {
+    echo "Cleaning up..."
+    docker stop indexer-test-db >/dev/null 2>&1
+    docker rm indexer-test-db >/dev/null 2>&1
+}
+
+# Register the cleanup function to run on script exit
+trap cleanup EXIT
+
+# Start PostgreSQL container with the same configuration as CI
+docker run -d \
+    --name indexer-test-db \
```

Comment:
Where are you getting indexer-test-db? Is there a Dockerfile for it?

Reply:
This is just the (arbitrary) container name, the image is `postgres:13`, i.e. https://hub.docker.com/_/postgres

Reply:
ah, my bad sorry